README.md

```diff
@@ -35,9 +35,10 @@ Asgardeo Auth React SDK  allows React applications to use OIDC or OAuth2 authent
 -   [Send HTTP Requests to Asgardeo](/API.md#httprequest)
 -   [Silent Sign In](/API.md#trysigninsilently)
 
-## Prerequisite
+## Prerequisite - Register your application in Asgardeo
 
-Create an organization in Asgardeo if you don't already have one. The organization name you choose will be referred to as `<org_name>` throughout this document.
+1. Register to [Asgardeo](https://wso2.com/asgardeo) and create an organization if you don't already have one. The organization name you choose will be referred to as `<org_name>` throughout this document.
+2. [Register a Single Page Application in Asgardeo](https://wso2.com/asgardeo/docs/guides/applications/register-single-page-app/#register-the-app) to obtain necessory keys to integrate your application with Asgardeo. You will obtain a `client-ID` from Asgardeo for your application which will need to embed later in your application for the integration. 
```

Comment:
Shall we use `<client_id>` here similar to `<org_name>`

Reply:
changed to client_ID